docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -26,34 +26,55 @@ The OIDC feature is currently unavailable on ACF2 systems.
 - [Troubleshooting](#troubleshooting)
 
 ## Usage
-
-The OIDC protocol is used by API ML client applications to verify the identity of a user with a distributed OIDC provider trusted by the mainframe security manager.
+API ML functions as an OIDC client application, enabling users to initiate the OIDC authentication flow
 After successful user login, the OIDC provider grants the client application a JWT Access Token along with an (JWT) Identity Token.
-The client application can pass this Access Token with subsequent requests to mainframe services routed through the API ML Gateway.
+Access token is then returned to the user agent in the "apimlAuthenticationToken" cookie.
+The user agent can pass this Access Token with subsequent requests to mainframe services routed through the API ML Gateway.
 The API ML Gateway then validates the OIDC Access Token. If the token is valid, the user identity from that token is mapped to the mainframe identity of the user.
-The API ML Gateway can then create mainframe user credentials (JWT or a PassTicket) according to the service's authentication schema configuration.
+The API ML Gateway can then create mainframe user credentials (e.g. JWT, PassTicket) according to the service's authentication schema configuration.
```

Comment:
Here is missing the case that OIDC is provided even if mainframe credentials cannot be mapped.